src/Documentation~/4x4-matrices.md

```diff
@@ -0,0 +1,48 @@
+# 4×4 matrices
+
+To create a 4×4 transformation matrix, use the constructors in [`float4x4`](xref:Unity.Mathematics.float4x4) to assign one value to all elements of the matrix, or individually set all 16 elements directly:
+
+
+```c#
+// Unity Mathematics example
+void Build4x4UnityMathematics()
+{
+   var c0 = new float4(1.0f, 0.0f, 0.0f, 0.0f);
+   var c1 = new float4(0.0f, 1.0f, 0.0f, 0.0f);
+   var c2 = new float4(0.0f, 0.0f, 1.0f, 0.0f);
+   var c3 = new float4(0.0f, 0.0f, 0.0f, 1.0f);
+   var m = new float4x4(c0, c1, c2, c3);
+}
+```
+
+## Multiplying a 4×4 matrix
+
+Unity Mathematics defines the `*` operator differently to `UnityEngine`. The `*` operator for [`float4x4`](xref:Unity.Mathematics.float4x4) implements componentwise multiplication. If you multiply a `float4x4` of all 1s with 0.5 on the diagonal, you get back the half identity because the upper and lower triangles of the matrix are multiplied by the respective zero entries from `f4x4_HalfIdentity`:
+
+```c#
+// Unity Mathematics example
+void OperatorMultiply4x4UnityMathematics()
+{
+   float4x4 result = f4x4_Ones * f4x4_HalfIdentity;
+   // result:
+   // 0.5, 0.0, 0.0, 0.0,
+   // 0.0, 0.5, 0.0, 0.0,
+   // 0.0, 0.0, 0.5, 0.0,
+   // 0.0, 0.0, 0.0, 0.5
+}
+```
+
+## Multiplying a 4×4 matrix and a 4D vector
```

Comment:
```suggestion
## Multiplying a 4×4 matrix by a 4D vector
```

Reply:
Saying "Multiply by" here feels more correct, but feel free to disagree and ignore depending on dev opinion!